network/validator/authorized_sender_validator.go

```diff
@@ -145,3 +162,9 @@ func (av *AuthorizedSenderValidator) isAuthorizedSender(identity *flow.Identity,
 
 	return what, nil
 }
+
+func (av *AuthorizedSenderValidator) checkSlashingViolationsConsumerErr(err error) {
+	if err != nil {
+		av.log.Error().Err(err).Msg("failed to disseminate slashing violation on slashing violations consumer")
+	}
+}
```

Comment:
Same comment as:

Reply:
The error return from slashing violations consumer has been removed.